### ISSUE TITLE
Add a -slow-start flag to ramp up requests

### ISSUE THREAD

**Issue by klingerf** [BuoyantIO/slow_cooker#58]
When sending a lot of load at a target, it would be nice to have slow_cooker ramp up to it's total qps over the course of a minute or two, rather than just sending the full amount of load at the outset. This would give the target a chance to warmup.

**Comment by stevej** [comment]
Yeah, a warm-up phase is pretty critical for services that rely on GC or cache warming.
